### PR TITLE
Add Czech Translation (2)

### PR DESCRIPTION
**Pull request by MrJaroslavik**

### CODE REVIEW

File: translations.php [hunk 1]
```diff
@@ -332,14 +364,14 @@
...
 			'zh-hant' => '社交媒體',
+			'cs' => 'Sociální Média',
 		],
 
 		'footer-desc' => [
 			'en' => 'Miraheze Limited <br />' .
 				'A company limited by guarantee registered in England and Wales company no. 12323854  <br />' .
 				'Except where otherwise noted, the content of this site is licensed under a ' .
 				'<a href="https://creativecommons.org/licenses/by-sa/4.0/">Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)</a>.',
-
```
Comment (RhinosF1) [inline translations.php:342]: Why has this line disappeared?

File: translations.php [hunk 2]
```diff
@@ -31,6 +32,8 @@
 			'th' => 'คุณสมบัติ',
 			'zh-hans' => '特点',
 			'zh-hant' => '特點',
+			'cs' => 'Funkce',
+
```
Comment (RhinosF1) [inline translations.php:36]: Is there supposed to be an additional blank like?

Comment (MrJaroslavik) [reply translations.php:36]: Not, should not be, sorry.

**Comment by MrJaroslavik** [comment]
Any progress? :)

**Pull request merged**